changelog.txt

```diff
@@ -0,0 +1 @@
+* Modify RTDB emulator to publish triggers to the functions emulator
```

Comment:
let's add a note about the default rules being open now

Reply:
And, this is breaking, so do something like `*BREAKING*`